### ISSUE TITLE
Crashed in the dev environment because of java.io.FileNotFoundException: enderio:models/farm.tcn

### ISSUE THREAD

**Issue by xsun2001** [SleepyTrousers/EnderIO#4114]
I am sorry but I really don't know how to setup the dev environment with EnderIO. I cloned the master branch in your repository. Then I executed `setupDecompWorkspace` and `genIntellijRuns` tasks in the Intellij IDEA. But when I want to run the minecraft, it throws `net.minecraftforge.client.model.ModelFormatException`, which caused by `java.io.FileNotFoundException: enderio:models/farm.tcn`. I am very sorry to disturb you so many times.
Here is the log:https://pastebin.com/QDpAFQuH
```
[18:53:42] [Client thread/INFO] [EnderIO]: Updating config...
[18:53:44] [Client thread/ERROR] [FML]: Fatal errors were detected during the transition from INITIALIZATION to POSTINITIALIZATION. Loading cannot continue
[18:53:44] [Client thread/ERROR] [FML]: 
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
	UCHI	mcp{9.05} [Minecraft Coder Pack] (minecraft.jar) 
	UCHI	FML{7.10.99.99} [Forge Mod Loader] (forgeSrc-1.7.10-10.13.4.1448-1.7.10.jar) 
	UCHI	Forge{10.13.4.1448} [Minecraft Forge] (forgeSrc-1.7.10-10.13.4.1448-1.7.10.jar) 
	UCHI	NotEnoughItems{1.0.5.111} [Not Enough Items] (NotEnoughItems-1.7.10-1.0.5.111-dev.jar) 
	UCHI	CodeChickenCore{1.0.7.46} [CodeChicken Core] (minecraft.jar) 
	UCHI	endercore{1.7.10-0.2.0.35_beta} [EnderCore] (EnderCore-1.7.10-0.2.0.35_beta-dev.jar) 
	UCHI	Waila{1.5.10} [Waila] (Waila-1.5.10_1.7.10-dev.jar) 
	UCHE	EnderIO{@VERSION@} [Ender IO] (main) 
	UCHI	EE3{0.3.507} [Equivalent Exchange 3] (EquivalentExchange3-1.7.10-0.3.507-dev.jar) 
[18:53:44] [Client thread/ERROR] [FML]: The following problems were captured during this phase
[18:53:44] [Client thread/ERROR] [FML]: Caught exception from EnderIO
net.minecraftforge.client.model.ModelFormatException: IO Exception reading model format
	at net.minecraftforge.client.model.techne.TechneModel.<init>(TechneModel.java:67) ~[forgeSrc-1.7.10-10.13.4.1448-1.7.10.jar:?]
	at net.minecraftforge.client.model.techne.TechneModelLoader.loadInstance(TechneModelLoader.java:26) ~[forgeSrc-1.7.10-10.13.4.1448-1.7.10.jar:?]
	at com.enderio.core.client.render.TechneUtil.getModel(TechneUtil.java:229) ~[EnderCore-1.7.10-0.2.0.35_beta-dev.jar:?]
	at com.enderio.core.client.render.TechneModelRenderer.<init>(TechneModelRenderer.java:28) ~[EnderCore-1.7.10-0.2.0.35_beta-dev.jar:?]
	at crazypants.enderio.machine.TechneMachineRenderer.<init>(TechneMachineRenderer.java:26) ~[main/:?]
	at crazypants.enderio.machine.farm.FarmingStationRenderer.<init>(FarmingStationRenderer.java:20) ~[main/:?]
	at crazypants.enderio.ClientProxy.load(ClientProxy.java:263) ~[main/:?]
	at crazypants.enderio.EnderIO.load(EnderIO.java:621) ~[main/:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_112]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_112]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_112]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_112]
	at cpw.mods.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:532) ~[forgeSrc-1.7.10-10.13.4.1448-1.7.10.jar:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_112]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_112]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_112]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_112]
	at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74) ~[guava-17.0.jar:?]
	at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47) ~[guava-17.0.jar:?]
	at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322) ~[guava-17.0.jar:?]
	at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304) ~[guava-17.0.jar:?]
	at com.google.common.eventbus.EventBus.post(EventBus.java:275) ~[guava-17.0.jar:?]
	at cpw.mods.fml.common.LoadController.sendEventToModContainer(LoadController.java:212) ~[forgeSrc-1.7.10-10.13.4.1448-1.7.10.jar:?]
	at cpw.mods.fml.common.LoadController.propogateStateMessage(LoadController.java:190) ~[forgeSrc-1.7.10-10.13.4.1448-1.7.10.jar:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_112]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_112]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_112]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_112]
	at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74) ~[guava-17.0.jar:?]
	at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47) ~[guava-17.0.jar:?]
	at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322) ~[guava-17.0.jar:?]
	at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304) ~[guava-17.0.jar:?]
	at com.google.common.eventbus.EventBus.post(EventBus.java:275) ~[guava-17.0.jar:?]
	at cpw.mods.fml.common.LoadController.distributeStateMessage(LoadController.java:119) [LoadController.class:?]
	at cpw.mods.fml.common.Loader.initializeMods(Loader.java:737) [Loader.class:?]
	at cpw.mods.fml.client.FMLClientHandler.finishMinecraftLoading(FMLClientHandler.java:311) [FMLClientHandler.class:?]
	at net.minecraft.client.Minecraft.startGame(Minecraft.java:597) [Minecraft.class:?]
	at net.minecraft.client.Minecraft.run(Minecraft.java:942) [Minecraft.class:?]
	at net.minecraft.client.main.Main.main(Main.java:164) [Main.class:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_112]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_112]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_112]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_112]
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135) [launchwrapper-1.11.jar:?]
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28) [launchwrapper-1.11.jar:?]
	at net.minecraftforge.gradle.GradleStartCommon.launch(Unknown Source) [start/:?]
	at GradleStart.main(Unknown Source) [start/:?]
Caused by: java.io.FileNotFoundException: enderio:models/farm.tcn
	at net.minecraft.client.resources.SimpleReloadableResourceManager.getResource(SimpleReloadableResourceManager.java:71) ~[forgeSrc-1.7.10-10.13.4.1448-1.7.10.jar:?]
	at net.minecraftforge.client.model.techne.TechneModel.<init>(TechneModel.java:62) ~[forgeSrc-1.7.10-10.13.4.1448-1.7.10.jar:?]
	... 46 more
[18:53:44] [Client thread/INFO] [STDOUT]: [net.minecraft.client.Minecraft:displayCrashReport:388]: ---- Minecraft Crash Report ----
// Uh... Did I do that?

Time: 17-3-28 下午6:53
Description: Initializing game

net.minecraftforge.client.model.ModelFormatException: IO Exception reading model format
	at net.minecraftforge.client.model.techne.TechneModel.<init>(TechneModel.java:67)
	at net.minecraftforge.client.model.techne.TechneModelLoader.loadInstance(TechneModelLoader.java:26)
	at com.enderio.core.client.render.TechneUtil.getModel(TechneUtil.java:229)
	at com.enderio.core.client.render.TechneModelRenderer.<init>(TechneModelRenderer.java:28)
	at crazypants.enderio.machine.TechneMachineRenderer.<init>(TechneMachineRenderer.java:26)
	at crazypants.enderio.machine.farm.FarmingStationRenderer.<init>(FarmingStationRenderer.java:20)
	at crazypants.enderio.ClientProxy.load(ClientProxy.java:263)
	at crazypants.enderio.EnderIO.load(EnderIO.java:621)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at cpw.mods.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:532)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
	at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
	at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
	at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
	at com.google.common.eventbus.EventBus.post(EventBus.java:275)
	at cpw.mods.fml.common.LoadController.sendEventToModContainer(LoadController.java:212)
	at cpw.mods.fml.common.LoadController.propogateStateMessage(LoadController.java:190)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
	at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
	at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
	at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
	at com.google.common.eventbus.EventBus.post(EventBus.java:275)
	at cpw.mods.fml.common.LoadController.distributeStateMessage(LoadController.java:119)
	at cpw.mods.fml.common.Loader.initializeMods(Loader.java:737)
	at cpw.mods.fml.client.FMLClientHandler.finishMinecraftLoading(FMLClientHandler.java:311)
	at net.minecraft.client.Minecraft.startGame(Minecraft.java:597)
	at net.minecraft.client.Minecraft.run(Minecraft.java:942)
	at net.minecraft.client.main.Main.main(Main.java:164)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
	at net.minecraftforge.gradle.GradleStartCommon.launch(Unknown Source)
	at GradleStart.main(Unknown Source)
Caused by: java.io.FileNotFoundException: enderio:models/farm.tcn
	at net.minecraft.client.resources.SimpleReloadableResourceManager.getResource(SimpleReloadableResourceManager.java:71)
	at net.minecraftforge.client.model.techne.TechneModel.<init>(TechneModel.java:62)
	... 46 more


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Stacktrace:
	at net.minecraftforge.client.model.techne.TechneModel.<init>(TechneModel.java:67)
	at net.minecraftforge.client.model.techne.TechneModelLoader.loadInstance(TechneModelLoader.java:26)
	at com.enderio.core.client.render.TechneUtil.getModel(TechneUtil.java:229)
	at com.enderio.core.client.render.TechneModelRenderer.<init>(TechneModelRenderer.java:28)
	at crazypants.enderio.machine.TechneMachineRenderer.<init>(TechneMachineRenderer.java:26)
	at crazypants.enderio.machine.farm.FarmingStationRenderer.<init>(FarmingStationRenderer.java:20)
	at crazypants.enderio.ClientProxy.load(ClientProxy.java:263)
	at crazypants.enderio.EnderIO.load(EnderIO.java:621)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at cpw.mods.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:532)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
	at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
	at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
	at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
	at com.google.common.eventbus.EventBus.post(EventBus.java:275)
	at cpw.mods.fml.common.LoadController.sendEventToModContainer(LoadController.java:212)
	at cpw.mods.fml.common.LoadController.propogateStateMessage(LoadController.java:190)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
	at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
	at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
	at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
	at com.google.common.eventbus.EventBus.post(EventBus.java:275)
	at cpw.mods.fml.common.LoadController.distributeStateMessage(LoadController.java:119)
	at cpw.mods.fml.common.Loader.initializeMods(Loader.java:737)
	at cpw.mods.fml.client.FMLClientHandler.finishMinecraftLoading(FMLClientHandler.java:311)
	at net.minecraft.client.Minecraft.startGame(Minecraft.java:597)

-- Initialization --
Details:
Stacktrace:
	at net.minecraft.client.Minecraft.run(Minecraft.java:942)
	at net.minecraft.client.main.Main.main(Main.java:164)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
	at net.minecraftforge.gradle.GradleStartCommon.launch(Unknown Source)
	at GradleStart.main(Unknown Source)

-- System Details --
Details:
	Minecraft Version: 1.7.10
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_112, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 667799776 bytes (636 MB) / 1038876672 bytes (990 MB) up to 1038876672 bytes (990 MB)
	JVM Flags: 3 total; -Xincgc -Xmx1024M -Xms1024M
	AABB Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
	FML: MCP v9.05 FML v7.10.99.99 Minecraft Forge 10.13.4.1448 9 mods loaded, 9 mods active
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
	UCHI	mcp{9.05} [Minecraft Coder Pack] (minecraft.jar) 
	UCHI	FML{7.10.99.99} [Forge Mod Loader] (forgeSrc-1.7.10-10.13.4.1448-1.7.10.jar) 
	UCHI	Forge{10.13.4.1448} [Minecraft Forge] (forgeSrc-1.7.10-10.13.4.1448-1.7.10.jar) 
	UCHI	NotEnoughItems{1.0.5.111} [Not Enough Items] (NotEnoughItems-1.7.10-1.0.5.111-dev.jar) 
	UCHI	CodeChickenCore{1.0.7.46} [CodeChicken Core] (minecraft.jar) 
	UCHI	endercore{1.7.10-0.2.0.35_beta} [EnderCore] (EnderCore-1.7.10-0.2.0.35_beta-dev.jar) 
	UCHI	Waila{1.5.10} [Waila] (Waila-1.5.10_1.7.10-dev.jar) 
	UCHE	EnderIO{@VERSION@} [Ender IO] (main) 
	UCHI	EE3{0.3.507} [Equivalent Exchange 3] (EquivalentExchange3-1.7.10-0.3.507-dev.jar) 
	GL info: ' Vendor: 'Intel' Version: '4.4.0 - Build 20.19.15.4474' Renderer: 'Intel(R) HD Graphics 5500'
	EnderIO: Found the following problem(s) with your installation:
                  * Our API got loaded from main. That's unexpected.
                  * Our API got loaded from main. That's unexpected.
                  * Our API got loaded from main. That's unexpected.
                  * Our API got loaded from main. That's unexpected.
                 This may have caused the error. Try reproducing the crash WITHOUT this/these mod(s) before reporting it.
	Launched Version: 1.7.10
	LWJGL: 2.9.1
	OpenGL: Intel(R) HD Graphics 5500 GL version 4.4.0 - Build 20.19.15.4474, Intel
	GL Caps: Using GL 1.3 multitexturing.
Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
Anisotropic filtering is supported and maximum anisotropy is 16.
Shaders are available because OpenGL 2.1 is supported.

	Is Modded: Definitely; Client brand changed to 'fml,forge'
	Type: Client (map_client.txt)
	Resource Packs: []
	Current Language: English (US)
	Profiler Position: N/A (disabled)
	Vec3 Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	Anisotropic Filtering: Off (1)
```

**Comment by HenryLoenwind** [comment]
Intellij? Sorry, no idea. I'm using Eclipse.

**Comment by xsun2001** [comment]
OK,I will try it on Eclipse.

**Comment by HenryLoenwind** [comment]
In that case:

`gradlew setupDecompWorkspace eclipse` and then opening the `eclipse` directory as workspace in Eclipse should work.

**Comment by xsun2001** [comment]
That is amazing!!! Eclipse works fine. But I am very confuse for this strange scene. I like Intellij and I want to know the reason of this problem. Is Intellij's way to process resources different to that of eclipse??

**Comment by tterrag1098** [comment]
You should probably update your IntelliJ, or maybe use a newer version of Minecraft. There was a bug *ages* ago where IntelliJ didn't load resources.

**Comment by xsun2001** [comment]
I know what I did wrong!! I forgot add `idea { module { inheritOutputDirs = true } }` at the end of build.gradle. I am really stupid and careless. Thank you for your help again! Now I will try to fix the bug in the version of 1.7.10.
![image](https://cloud.githubusercontent.com/assets/14010993/24450710/78ea9c22-14ae-11e7-87f0-34457ee6d366.png)
HAPPY!
